app/api/schema/tickets.py

```diff
@@ -78,6 +83,8 @@ def validate_discount_code(self, data, original_data):
     description = fields.Str(allow_none=True)
     type = fields.Str(required=True)
     price = fields.Float(validate=lambda n: n >= 0, allow_none=True)
+    min_price = fields.Float(validate=lambda n: n >= 0, allow_none=True)
```

Comment:
if nullable is false, why would you allow none here?

Reply:
Yeah, noticed that. I'm updating it right now.